### ISSUE TITLE
unsupported operand type(s) for +: 'datetime.datetime' and 'int'

### ISSUE THREAD

**Issue by amogh-talpallikar** [James1345/django-rest-knox#10]
AuthToken.objects.create function tries to add expires as integer to datetime resulting in the above exception.  It should be wrapped as a timedelta instead.

http://james1345.github.io/django-rest-knox/settings/

Here in the doc you do mention that TOKEN_TTL should be timedelta but then the example settings show it to be an integer. Either the doc should be fixed or you should support integer as hours as also mentioned in the doc.

```
  REST_KNOX = {
    'SECURE_HASH_ALGORITHM': 'cryptography.hazmat.primitives.hashes.SHA512',
    'AUTH_TOKEN_CHARACTER_LENGTH': 64,
    'TOKEN_TTL': 10,
    'USER_SERIALIZER': 'knox.serializers.UserSerializer',
 }
```

**Comment by James1345** [comment]
You are correct, the example is wrong. Please use timedelta, the doc shall be corrected.

**Comment by wiesson** [comment]
Please update the documentation.

**Comment by belugame** [comment]
this was fixed in c928368c
